### PR TITLE
chore(refactor): Small code finding refactorings

### PR DESCRIPTION
**Pull request by sbuettner**
## Description

Small refactorings after looking at some existing code.

### CODE REVIEW

File: connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java [hunk 1]
```diff
@@ -54,14 +54,14 @@ public static Map<String, Object> createOutputVariables(
...
 
-    Optional.ofNullable(resultExpression)
-        .filter(s -> !s.isBlank())
-        .map(
-            expression ->
-                FEEL_ENGINE_WRAPPER.evaluateToJson(
-                    expression, responseContent, wrapResponse(responseContent)))
-        .map(json -> parseJsonVarsAsTypeOrThrow(json, Map.class, resultExpression))
-        .ifPresent(outputVariables::putAll);
+    if (resultExpression != null && !resultExpression.isBlank()) {
```
Comment (johnBgood) [inline connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java:57]:
```suggestion
    if (StringUtils.isNotBlank(resultExpression)) {
```

**Review by johnBgood — DISMISSED** [review]

File: connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java (hunk 1 above)
Comment (sbuettner) [reply connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java:57]: @johnBgood What is the benefit over: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank() ?

Comment (johnBgood) [reply connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java:57]: the `null` check, and conciseness I guess

**Review by johnBgood — APPROVED** [review]
💯

**Pull request merged**